osbuild/libdir.py

```diff
@@ -0,0 +1,37 @@
+import os
+from typing import List
+
+
+class Libdir:
+    """Libdir abstracts the osbuild --libdir handling"""
+
+    def __init__(self, libdir: str) -> None:
+        self._libdir = ":".join(os.path.abspath(p)
```

Comment:
I don't entirely grok this. We split, then join, then assign a string then start splitting that string again in the property below. Would it be easier to assign the list instead?

Reply:
Yeah, good point. Fixed.